docs/change_log/release-3.3.md

```diff
@@ -87,6 +87,9 @@ The following new features have been included in the 3.3 release:
   `markdown.extensions.headerid.slugify_unicode` to the `slugify` configuration option
   to use the new behavior.
 
+* A table of contents level class has been added to the [Table of Contents](../extensions/toc.md)
+  extension, e.g. class="toc1". This allows for individual styling for each table of contents level.
+
```

Comment:
Version 3.3 has already been released so any new changes should not be documented here. As this is a new feature, it would need to be in a new release (presumably 3.4). The appropriate file will need to be added along with an entry in the [change log][1] which links to it.

[1]: https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/index.md